projects/ad9081_fmca_ebz/vck190/system_bd.tcl

```diff
@@ -18,12 +18,25 @@ set ADI_PHY_SEL 0
 source $ad_hdl_dir/projects/ad9081_fmca_ebz/common/ad9081_fmca_ebz_bd.tcl
 source $ad_hdl_dir/projects/scripts/adi_pd.tcl
 
-set mem_init_sys_path [get_env_param ADI_PROJECT_DIR ""]mem_init_sys.txt;
-
 #system ID
-ad_ip_parameter axi_sysid_0 CONFIG.ROM_ADDR_BITS 9
-ad_ip_parameter rom_sys_0 CONFIG.PATH_TO_FILE "[pwd]/$mem_init_sys_path"
-ad_ip_parameter rom_sys_0 CONFIG.ROM_ADDR_BITS 9
-
-sysid_gen_sys_init_file
-
+ad_ip_parameter axi_sysid_0 CONFIG.ROM_ADDR_BITS 10
+ad_ip_parameter rom_sys_0 CONFIG.PATH_TO_FILE "$mem_init_sys_file_path/mem_init_sys.txt"
+ad_ip_parameter rom_sys_0 CONFIG.ROM_ADDR_BITS 10
+
+set sys_cstring "$ad_project_params(JESD_MODE)\
```

Comment:
Add `"JESD_MODE=$...` to know that the first value is from this parameter.
Also, `REF_CLK_RATE` parameter is missing.

Reply:
The parameters string gets pretty long for the JESD projects and knowing that the mode is always going to be "8b10b" or "64b66b" I chose to skip the "MODE" name to make it a little bit shorter because I thought it was pretty obvious what is referring to, but I can add it to all of the projects if you want to.

Reply:
Yes, I didn't think of the actual string that will be in JESD_MODE. Good point.